### ISSUE TITLE
Ability to change how YAML::Null is emitted

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jbeder/yaml-cpp#137]
```
What steps will reproduce the problem?
1. emmiter << YAML::Null

What is the expected output? What do you see instead?

{{{
~
}}}

What version of the product are you using? On what operating system?
0.2.7

Please provide any additional information below.

It'd be nice if you could modify how YAML::Null is represented, I have a file I 
have to generate that way:

{{{
- foo: 123
- bar:
- baz: "yeah"
}}}

To mean, "no values" for "bar"... right now this emits "- bar: ~".
```

Original issue reported on code.google.com by `philippe...@gmail.com` on 29 Nov 2011 at 4:16

**Comment by outofthecave** [comment]
It would be nice if one could choose between all valid representations of `null` in YAML, i.e. `~`, `null`, `Null`, `NULL`, or no output, see http://yaml.org/type/null.html.

**Comment by jbeder** [comment]
Fixed #906.
